### PR TITLE
Feature/reusable-button-component

### PR DESCRIPTION
**Pull request by Marcelluki**
## Describe your changes
- Created Button.tsx for Reusable button component
- added Button.css for styling
- Button accepts variants 

## Issue ticket number and link
https://tripleten-apiary.atlassian.net/browse/EES-11
EES-11

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] If there are changes to components, I have added / updated automated tests
- [ ] I have made any necessary updates to Storybook to accompany these changes
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: src/main.tsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import { ApolloClient, ApolloProvider, InMemoryCache } from "@apollo/client";
 import React from "react";
 import ReactDOM from "react-dom/client";
-import { BrowserRouter, Route, Routes } from "react-router-dom";
+import { BrowserRouter } from "react-router-dom";
 import "./global/default.css";
+import { Button } from "./components/Button";
 
 const client = new ApolloClient({
   uri: "http://localhost:8080/api/graphql",
```
Comment (RPinkha) [inline src/main.tsx:9]: URL is misspelled.

File: src/main.tsx [hunk 2]
```diff
@@ -13,9 +14,8 @@ ReactDOM.createRoot(document.getElementById("root")!).render(
   <React.StrictMode>
     <ApolloProvider client={client}>
       <BrowserRouter>
-        <Routes>
-          <Route path="/" element={<div>Home Route</div>} />
-        </Routes>
+        
+        <Button submit={console.log} buttonText={"Contact Us!"}/>
```
Comment (RPinkha) [inline src/main.tsx]: The button text doesn't seem to be reusable

**Review by RPinkha — CHANGES_REQUESTED** [review]
No tailwind used. 
No button.css for styling that can't be achieved with tailwind.

File: src/main.tsx [hunk 3]
```diff
@@ -1,8 +1,9 @@
 import { ApolloClient, ApolloProvider, InMemoryCache } from "@apollo/client";
 import React from "react";
 import ReactDOM from "react-dom/client";
-import { BrowserRouter, Route, Routes } from "react-router-dom";
+import { BrowserRouter } from "react-router-dom";
 import "./global/default.css";
+import { Button } from "./components/Button/Button";
```
Comment (XDRO) [inline src/main.tsx:6]: There is still an issue with Github tests, please ensure that the './components/Button/Button is the correct path.

Comment (XDRO) [reply src/main.tsx:6]: I notice that there is not a Button folder, file, or component when I switch to this branch. As it is not present in the repo on Github.

**Review by XDRO — CHANGES_REQUESTED** [review]
You must ensure that you have the Button component uploaded to Github. You are still failing Github test, from line 6.

**Pull request closed without merging**